db/mongoose.js

```diff
@@ -1,13 +1,35 @@
 const mongoose = require("mongoose");
 const keys = require("../config/keys");
-mongoose.connect(
-  keys.mongoURI,
-  { useNewUrlParser: true, useUnifiedTopology: true, useFindAndModify: false, useCreateIndex: true },
-  function (err) {
-    if (err) {
-      console.log(err);
-    } else {
-      console.log("MongoDB Connected");
-    }
-  }
-);
+mongoose
+  .connect(keys.mongoURI, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+    dbName: "Messportal",
```

Comment:
I guess we can leave the DB name to be configured with the help of MONGO_URI, what do you think?

Reply:
Its just about preference nothing else.

Reply:
Can you then just update the PR to remove this configuration? Rest looks good to me.